docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -289,6 +294,84 @@ spec:
 ** Go to the pipeline run with *Build* in the *Type* column and confirm that the `prefetch-dependencies` stage displays a green checkmark. This indicates that the build process successfully fetched all dependencies.
 * From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
 
+== Enabling prefetch builds for `rpm`
+
+Cachi2 has a dev-preview package manager capable of fetching `rpm` dependencies. This requires the use of a pair of `rpms.in.yaml` and `rpms.lock.yaml` files to be committed to your repository. You write a `rpms.in.yaml` file and the link:https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#what-is-this[rpm-lockfile-prototype] CLI tool resolves that to produce a `rpms.lock.yaml` file. Cachi2 fetches those specific rpms and enables your build to install them without network access.
+
+.Prerequisites
+* You have an up-to-date installation of link:https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#installation[rpm-lockfile-prototype].
+
+.Procedure
+
+To prefetch dependencies for a component build, complete the following steps:
+
+. Create a `rpms.in.yaml` file in your git repository, with the following contents:
+
++
+[source,yaml]
+----
+packages: [nethack] <1>
+contentOrigin:
+  repofiles: ["./fedora.repo"] <2>
+----
+<1> The `*packages*` list is the list of packages you want to install in your Container. Only include first-order dependencies here, i.e. the packages you want to install.
+<2> This should be a reference to a repo file, like those found in `/etc/yum.repos.d/`. This tells the Cachi2 where to find your rpm and its dependencies.
+
+. Copy any necessary yum/dnf repo files into your git repository. If you are using a fedora base image, that looks like:
```

Comment:
This is the base image of the container that you are building? Would users know how to select the appropriate file(s) for rhel/UBI from this?

Reply:
Hm, let's provide a command to extract it from the base image they're working with.

My favorite command for that is `oc image extract ...` though, and I don't know an "easy" alternative with skopeo.

Reply:
Well, I guess this works:

```
podman run -it registry.redhat.io/ubi9/ubi:latest cat /etc/yum.repos.d/ubi.repo > ubi.repo
```

Reply:
The *.repo files don't really need to be commited to the user's repo, they only need to be there for the lockfile resolution (although it doesn't hurt to commit them). Not sure if we should point that out, or if that'll only make it more confusing.

Reply:
I am similarly unsure if one is more confusing than the other.

I guess, without more user reactions to go on, I'd assume that committing the file is more explicit and clear. FWIW, it kind of looks like a beefier version of content_sets.yaml (some openstack devs today understood it right away when I tested out explaining it that way).